samples/zigbee/shell/prj.conf

```diff
@@ -54,3 +54,4 @@ CONFIG_ZIGBEE_SHELL_ENDPOINT=64
 
 CONFIG_LOG_CMDS=y
 CONFIG_LOG_STRDUP_MAX_STRING=370
+CONFIG_LOG_STRDUP_BUF_COUNT=24
```

Comment:
Does it affect only the shell sample or all applications, that would like to use the Zigbee shell component?

Reply:
Most likely shell sample only because `CONFIG_LOG_MINIMAL=n`.
I haven't seen similar problems with samples extended with ZIgbee shell library. Need to check to confirm.

Reply:
I stress-tested ZIgbee Coordinator sample with Zigbee Shell enabled and haven't seen single log_strdup() fail.
Other samples also shouldn't be affected by that, only Zigbee Shell sample.